nss-day-cohort-46/rare-client-clever-centaurs
# Kb two client

# Description
- added addPost and updatePost
- added add post/ create post form

 
## Type of change

# Testing Instructions
```
git fetch --all
git checkout kb-twoClient
```
be sure you can add a post on the app
check to see that you can edit a post in postman

# Checklist:
